libs/dh/shared/test-util-auth/src/lib/local-storage-fake.service.ts

```diff
@@ -0,0 +1,49 @@
+/**
+ * @license
+ * Copyright 2020 Energinet DataHub A/S
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License2");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+export class LocalStorageFake implements Storage {
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  [name: string]: any;
+
+  private internalLength = 0;
+
+  public get length() {
+    return this.internalLength;
+  }
+
+  public readonly setItem = (key: string, value: string) => {
+    this[key] = value;
+    this.internalLength = Object.keys(this).length;
+  };
+
+  public readonly getItem = (key: string) => {
+    return this[key];
+  };
+
+  public readonly removeItem = (key: string) => {
+    delete this[key];
```

Comment:
issue: this will also delete one of the methods (e.g. `getItem` or `setItem`) if sent by mistake.

Reply:
Fixed. ScopeStorage no longer has a public removeItem method.